standard-covoiturage_openapi.yaml

```diff
@@ -584,11 +770,9 @@ components:
       - passengerDropLat
       - passengerDropLng
       - status
-      - webUrl
```

Comment:
If marked as optional here, it must be clear that it is mandatory for the booking events flow (https://github.com/fabmob/standard-covoiturage/pull/5).